package.json

```diff
@@ -1,37 +1,111 @@
 {
   "name": "telemetry-github",
-  "version": "0.0.13",
+  "version": "0.3.2",
   "description": "telemetry for GitHub client apps",
-  "main": "./out/index.js",
+  "repository": {
+    "url": "https://github.com/atom/telemetry"
+  },
+  "main": "./lib/index.js",
+  "types": "./index.d.ts",
   "scripts": {
-    "mocha": "./node_modules/.bin/mocha --require mock-local-storage --compilers ts:ts-node/register ./test/*.spec.ts",
-    "test": "npm run mocha && npm run lint",
-    "lint": "tslint -c tslint.json ./src/*.ts ./test/*.ts",
-    "prepare": "tsc"
+    "info": "npm-scripts-info",
+    "build": "run-s clean build:main",
+    "build:main": "tsc -p src/",
+    "watch": "run-s clean \"build:main -- -w\" \"test:unit -- --watch\"",
+    "test": "run-s build test:*",
+    "test:unit": "ts-mocha -p src/tsconfig.json --require mock-local-storage --require async-dump.js test/*.spec.ts",
```

Comment:
it would be nice to have one command for running test and lint :-) otherwise I tend to forget to fix linter problems and it wastes my time